### ISSUE TITLE
Preset concept should be moved to the "module" bounded context

### ISSUE THREAD

**Issue by murdos** [jhipster/jhipster-lite#10651]
A `preset` is collection of preselected modules that could be applied together, so it rather belong to the "module" bounded context.

It actually resides in the "project" bounded context, whereas this bounded context deals with an existing generated project: history of applied modules, download of the project, ...

**Comment by renanfranca** [comment]
I agree. I thought about that while I was doing it. I decided on my last commit to where it should go 😅

So it's going to be at the same place as the list of modules,  isn't it?

**Comment by murdos** [comment]
> I agree. I thought about that while I was doing it. I decided on my last commit to where it should go 😅

😅 I also did not pay attention to that when I've reviewed the code. I saw that accidentally while looking for the best location for the automatic cleanup of generated projects (#10575)

> So it's going to be at the same place as the list of modules,  isn't it? 

I think it makes more sense. Alongside landscape stuff.
